src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -95,6 +95,8 @@ public partial class DynamoView : Window, IDisposable
 
         private FileTrustWarning fileTrustWarningPopup = null;
 
+        public ShortcutToolbar ShortcutBar { get { return shortcutBar; } }
```

Comment:
I think `internal` should be sufficient, if needed for test, we can make it internally visible

Reply:
I set it to public because it's being used in the notifications extension

Reply:
Any reason for not making this internal like Aaron suggested?

Reply:
I'm using it at the notification extension. I didn't think any alternative for it :/